modules/claims_api/app/services/claims_api/disability_compensation/docker_container_service.rb

```diff
@@ -25,6 +25,10 @@ def upload(claim_id)
                              "Successfully submitted to Docker container with response: #{evss_res}")
         # update with the evss_id returned
         auto_claim.update!(evss_id: evss_res[:claimId])
+      rescue => e
+        auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
+        auto_claim.evss_response = e.errors if e.methods.include?(:errors)
```

Comment:
Nit: I think `respond_to?` here would be more inline with what we do in all these spots.